azext_iot/digitaltwins/commands_jobs.py

```diff
@@ -0,0 +1,40 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.digitaltwins.providers.import_job import ImportJobProvider
+from knack.log import get_logger
+from uuid import uuid4
+
+logger = get_logger(__name__)
+
+
+def create_import_job(
+    cmd, name_or_hostname, data_file_name, input_blob_container_name, input_storage_account_name,
+    output_file_name, output_blob_container_name=None, output_storage_account_name=None, resource_group_name=None
+):
+    import_job_provider = ImportJobProvider(cmd=cmd, name=name_or_hostname, rg=resource_group_name)
+    job_id = "bulk-import-job-{}".format(str(uuid4()).replace("-", ""))
```

Comment:
Is there a reason for the job id to start with bulk-import-job-? Can we just have the uuid4? what if the user wants to specify a job name?

Reply:
The **bulk-import-job-** prefix allows users to understand that it is the import job id, instead of a random guid. There is no concept of job name, only job id which is used by the system and not something user cares about, also it always needs to be unique, hence best to be system generated.